apimonitor/tests/test_health.py

```
@@ -0,0 +1,191 @@
+            # Verify headers were passed
+            mock_client.request.assert_called_with(
+                method='GET',
+                url='https://api.example.com/health',
+                headers={'X-API-Key': 'test-key'},
+                follow_redirects=True
+            )
+            # Also ensure httpx.Client was created with TLS verify enabled
+            _, client_kwargs = mock_client_class.call_args
+            assert client_kwargs.get('verify') is True
+            assert client_kwargs.get('timeout') is not None
+            assert result['ok'] is True
```

Ruff (0.12.2)

Undefined name `result`

(F821)